src/viam/components/arm/arm.py

```diff
@@ -106,3 +106,17 @@ async def is_moving(self) -> bool:
             bool: Whether the arm is moving.
         """
         ...
+
+    @abc.abstractmethod
+    async def get_kinematics(self, *, timeout: Optional[float] = None) -> Tuple[KinematicsFileFormat.ValueType, bytes]:
+        """
+        Get the kinematics information associated with the arm.
+
+        Returns:
+            Tuple[KinematicsFileFormat.Valuetype, bytes]:
+            (
+                format: The kinematics of the component in URDF format or Viam's kinematic parameter format (spatial vector algebra),
+                bytes: The byte contents of the file
+            )
```

Comment:
I think I prefer
```suggestion
        Returns:
            Tuple[KinematicsFileFormat.ValueType, bytes]:

                - KinematicsFileFormat.ValueType:
                  The format of the file, either in URDF format or Viam's kinematic parameter format (spatial vector algebra).

                - bytes:
                  The byte contents of the file.
```
which looks like 
![Screen Shot 2023-06-29 at 17 40 59 PM](https://github.com/viamrobotics/viam-python-sdk/assets/90270663/5be7dc03-3233-472d-87e7-96ec0456e4f8)
 let me know if you think a different format will look better!